src/query/settings/src/lib.rs

```diff
@@ -354,6 +354,17 @@ impl Settings {
                 desc: "The maximum query execution time. it means no limit if the value is zero. default value: 0.",
                 possible_values: None,
             },
+            #[cfg(feature = "hive")]
+            SettingValue {
```

Comment:
Why need a setting to control push down or not?

Reply:
if pushdown is not work as expected, we may lost data.  disable pushdown is one way to analyse the problem.